CommandHelper/LocalPackages/commands/chars/amazonwarrior.ms

```diff
@@ -0,0 +1,37 @@
+maxHearts: 20
+buffs:
+- /effect <plr> 5 9999 5 true
+- /effect <plr> 10 9999 1 true
+- /effect <plr> 21 9999 4 true
+- /give <plr> 420 1 16:5 name:&6Am&7az&eon&8ia&6n_&eLasso
+- /give <plr> minecraft:iron_sword 1 16:5 name:&6Am&7az&eon&8ia&6n_&eSword
+- /give <plr> minecraft:bow 1 50:1 51:1 name:&6Am&7az&eon&8ia&6n_&eBow
+- /give <plr> minecraft:arrow 10
+- /give <plr> minecraft:shield 1 0 name:&6Am&7az&eon&8ia&6n_&eShield
+- /give <plr> minecraft:iron_helmet 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eHe&6lm&7et
+- /give <plr> minecraft:iron_chestplate 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eCh&6es&7tpl&8ate
+- /give <plr> minecraft:iron_leggings 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eLe&7gg&6in&8gs
+- /give <plr> minecraft:iron_boots 1 0 0:10 1:10 3:10 4:10 7:10 name:&6Am&7az&eon&8ia&6n_&eBo&6o&7ts
```

Comment:
I cant edit

Reply:
There should be no reason why you can't edit, this code is literally under your account, which is why it has ur name by it. what error are you getting?